cmd/snap-update-ns/mount-entry.c

```diff
@@ -94,6 +104,37 @@ void sc_cleanup_mount_entry_list(struct sc_mount_entry **entryp)
 	}
 }
 
+int sc_compare_mount_entry(const struct sc_mount_entry *a,
+			   const struct sc_mount_entry *b)
+{
+	int result;
+	if (a == NULL || b == NULL) {
+		die("cannot compare NULL mount entry");
```

Comment:
No, we never have NULL entries that mean anything. We only have an empty list that is expressed by an NULL entry. The sorting routine will not compare NULL pointers.